conans/test/functional/command/profile_test.py

```diff
@@ -69,6 +74,10 @@ def show_test(self):
     def profile_update_and_get_test(self):
         client = TestClient()
         client.run("profile new ./MyProfile --detect")
+        if "WARNING: GCC OLD ABI COMPATIBILITY" in client.out:
+            self.assertIn("edit the ./MyProfile profile at", client.out)
+            self.assertIn("profiles/./MyProfile", client.out)
```

Comment:
Yes, @danimtb is right, this is wrong, as ``./MyProfile`` is a local file, not in the cache.

The only way to implement this would be to pass the path to the file, not the filename.

Reply:
The full message is:

```
Conan detected a GCC version > 5 but has adjusted the 'compiler.libcxx' setting to
'libstdc++' for backwards compatibility.
Your compiler is likely using the new CXX11 ABI by default (libstdc++11).

If you want Conan to use the new ABI, edit the {profile} profile at:

    ~/.conan/profiles/{profile}

adjusting 'compiler.libcxx=libstdc++11'
```

If I understand, we should print:

    ... edit the MyProfile profile at
    ~/.conan/profiles/./MyProfile

Is it correct?

Reply:
Not really. In that test, you are using a profile relative to the current dir, not to the "profiles" folder in the cache. The path showed should be either an absolute path in that case or a relative path to the current dir

Reply:
Understood. Thanks Dani!

Reply:
Done! Please @memsharded @danimtb review again when possible